tests/flow/async/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -356,13 +357,15 @@ function test3() {
 //
 
 function test4() {
-  async function voidoid5(): void { // error, void != Promise<void>
+  async function voidoid5(): void {
```

Comment:
The new behavior is reasonable, but might not play well with, eg; `eslint-disable-line some-rule`